src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.vb

```diff
@@ -72,12 +73,7 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             Get
                 If m_ControlData Is Nothing Then
 
-                    TargetFrameworkPropertyControlData = New TargetFrameworkPropertyControlData(
-                            VslangProj100.VsProjPropId100.VBPROJPROPID_TargetFrameworkMoniker, Const_TargetFrameworkMoniker,
-                            TargetFramework,
-                            AddressOf SetTargetFrameworkMoniker, AddressOf GetTargetFrameworkMoniker,
-                            ControlDataFlags.ProjectMayBeReloadedDuringPropertySet Or ControlDataFlags.NoOptimisticFileCheckout,
-                            New Control() {TargetFrameworkLabel})
+                    TargetFrameworkPropertyControlData = New TargetFrameworkPropertyControlData(VslangProj100.VsProjPropId100.VBPROJPROPID_TargetFrameworkMoniker, TargetFramework, AddressOf SetTargetFrameworkMoniker, AddressOf GetTargetFrameworkMoniker, ControlDataFlags.ProjectMayBeReloadedDuringPropertySet Or ControlDataFlags.NoOptimisticFileCheckout, New Control() {TargetFrameworkLabel})
```

Comment:
Is this just a formatting change?

Reply:
Formatting + deleted the `TargetFramework` name, as I changed the method's signature.